configure

```diff
@@ -12,7 +12,7 @@
 # Other settings: LTO, PGO?  Consider moving prefix, LTO, PGO to build and
 # install steps.
 
-TMP=${TMPDIR:-/tmp}  # Assume that any system has $TMP set or /tmp exists
+TMP=${TMPDIR:-/tmp}  # Assume that any system has $TMPDIR set or /tmp exists
```

Comment:
Hm I was wondering why you removed TMP.  But this answer seems to say that TMPDIR or /tmp should exist.

https://unix.stackexchange.com/questions/174817/finding-the-correct-tmp-dir-on-multiple-platforms

Reply:
Ah OK I see you mentioned that in the commit description.  Thanks.